### PR TITLE
[Elementwise] allow serialize OP_SCALAR_TRUE_DIV and OP_SCALAR_FLOOR_DIV in PCG

### PR DESCRIPTION
**Pull request by jiazhihao**

### CODE REVIEW

File: examples/python/keras/func_cifar10_alexnet.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def top_level_task():
     image = x_train[i, :, :, :]
     image = image.transpose(1, 2, 0)
     pil_image = Image.fromarray(image)
-    pil_image = pil_image.resize((229,229), Image.NEAREST)
+    pil_image = pil_image.resize((229,229), Image.Resampling.NEAREST)
```
Comment (lockshaw) [inline examples/python/keras/func_cifar10_alexnet.py:43]: What is this change for?

**Review by lockshaw — CHANGES_REQUESTED** [review]

File: src/ops/concat.cc [hunk 2]
```diff
@@ -270,7 +268,7 @@ void Concat::forward_task(Task const *task,
     // inputs[i] = helperGetTensorPointerRO<float>(
     //     regions[i + 1], task->regions[i + 1], FID_DATA, ctx, runtime);
     inputs[i] = helperGetGenericTensorAccessorRO(
-        DT_FLOAT, regions[0], task->regions[0], FID_DATA, ctx, runtime);
+        DT_FLOAT, regions[i + 1], task->regions[i + 1], FID_DATA, ctx, runtime);
```
[the remark this answers is not in the document]
Comment (jiazhihao) [reply src/ops/concat.cc:271]: They belong to #423.

[the remark this answers is not in the document]
Comment (jiazhihao) [reply src/ops/concat.cc:271]: Don't know why it still shows the diff.

File: examples/python/keras/func_cifar10_alexnet.py (hunk 1 above)
Comment (jiazhihao) [reply examples/python/keras/func_cifar10_alexnet.py:43]: They belong to #423.

Comment (jiazhihao) [reply examples/python/keras/func_cifar10_alexnet.py:43]: This is to resolve a warning from the pil_image python library

**Review by lockshaw — APPROVED** [review]

**Comment by jiazhihao** [comment]
closes #375

**Pull request merged**